Archs/MIPS/MipsOpcodes.cpp

```diff
@@ -135,6 +135,8 @@ const tMipsOpcode MipsOpcodes[] = {
 	{ "lv.s",	"vt,(s)",			MIPS_OP(0x32),			MA_PSP,		MO_VFPU_SINGLE|MO_VFPU_MIXED },
 	{ "lld",	"t,i16(s)",			MIPS_OP(0x34),			MA_MIPS3,	MO_64BIT|MO_DELAYRT },
 	{ "lld",	"t,(s)",			MIPS_OP(0x34),			MA_MIPS3,	MO_64BIT|MO_DELAYRT },
+	{ "ldc1",	"T,i16(s)",			MIPS_OP(0x35),			MA_MIPS2,	0 },
+	{ "ldc1",	"T,(s)",			MIPS_OP(0x35),			MA_MIPS2,	0 },
```

Comment:
Hm, interesting. I guess most of the custom CPUs for the other systems just omit them then, as I haven't seen them there before. Can you set the architecture to MA_MIPS2|MA_EXPS2|MA_EXPSP?